lib/screens/homepage.dart

```diff
@@ -271,24 +343,30 @@ class _HomePageContentState extends State<HomePageContent> {
             right: 15,
             left: 15,
             child: GestureDetector(
-              onTap: ()=>Navigator.push(context, MaterialPageRoute(builder: (context)=>SearchPage())),
+              onTap: () => Navigator.push(context,
+                  MaterialPageRoute(builder: (context) => SearchPage())),
               child: Container(
                 decoration: BoxDecoration(
                     color: Colors.white,
-                    borderRadius: BorderRadius.all(Radius.circular(10.0))
-                ),
+                    borderRadius: BorderRadius.all(Radius.circular(10.0))),
```

Comment:
Thank you for improving code readability

Reply:
I'm glad I could help.